libsolidity/analysis/FunctionCallGraph.cpp

```diff
@@ -165,6 +171,18 @@ bool FunctionCallGraphBuilder::visit(Identifier const& _identifier)
 
 bool FunctionCallGraphBuilder::visit(MemberAccess const& _memberAccess)
 {
+	TypePointer exprType = _memberAccess.expression().annotation().type;
+	ASTString const& memberName = _memberAccess.memberName();
+
+	if (auto magicType = dynamic_cast<MagicType const*>(exprType))
```

Comment:
For consistency, I'd either remove `exprType` and use `_memberAccess.expression().annotation().type` directly or use `exprType` also in the code below so that it's easier to see that both fragments are referring to the same thing.

```suggestion
	if (auto magicType = dynamic_cast<MagicType const*>(_memberAccess.expression().annotation().type))
```

Reply:
Added it to both